FSharp.AutoComplete/test/integration/ParamCompletion/ParamCompletionRunner.fsx

```diff
@@ -0,0 +1,30 @@
+#load "../TestHelpers.fsx"
+open TestHelpers
+open System.IO
+open System
+
+(*
+ * This test is a simple sanity check of a basic run of the program.
+ * A few completions, files and script.
+ *)
+
+Environment.CurrentDirectory <- __SOURCE_DIRECTORY__
+File.Delete "output.txt"
+
+let p = new FSharpAutoCompleteWrapper()
+
+p.send "outputmode json\n"
+p.project "Test1.fsproj"
+p.parse "FileTwo.fs"
+p.parse "Program.fs"
+Threading.Thread.Sleep(6000)
+p.methods "Program.fs" 4 36
+p.methods "Program.fs" 4 37
```

Comment:
This one is failing, but I think it shouldn't. Using 1-indexed columns this is a request at `$` in `let testval = FileTwo.NewObjectType($)`, so I think we should return results here. The previous one is actually on the open bracket, which it would be nice if it succeeded, but I think slightly less important than this one.